migrations/README.rst

```diff
@@ -0,0 +1,252 @@
+==================
+Database Migration
+==================
+
+Pepys-import repository uses `Alembic <https://github.com/alembic/alembic>`_ for data migration. Alembic is a database migration tool which is maintained by SQLAlchemy.
+It is possible to apply the existing migrations to the users' database via :code:`pepys_admin`.
+Don't forget to check your configuration file before running :code:`pepys_import` or :code:`pepys_admin`.
+
+Installation
+------------
+Alembic is in the requirements of the project. However, there is one post-installation step to run it without any problem. You should install the pepys-import project in an editable mode. The commands must be run are as follows:
+
```

Comment:
@BarisSari - for which groups of users are these steps necessary?  Just devs that make database schema changes?

Reply:
Yes, it's for the developers to run `alembic` commands

Reply:
Ok.  In that case I think we need to "switch together" the developer guidance.  Could you come up with a tidy way of including this guidance in the `Developer Setup` page?

Reply:
Sure @IanMayo . I moved this step to the Developer Guide's "Create Python virtual environment and install dependencies" section:

https://pepys-import--337.org.readthedocs.build/en/337/developer_guide.html#create-python-virtual-environment-and-install-dependencies